bitcoin/core/__init__.py

```diff
@@ -24,12 +24,15 @@
 
 # Core definitions
 COIN = 100000000
-MAX_MONEY = 21000000 * COIN
 MAX_BLOCK_SIZE = 1000000
 MAX_BLOCK_SIGOPS = MAX_BLOCK_SIZE/50
 
-def MoneyRange(nValue):
-    return 0 <= nValue <= MAX_MONEY
+def MoneyRange(nValue, useParams=None):
```

Comment:
I'd change this to params instead, to match the formatting that'll be used elsewhere.